src/activemasternode.cpp

```diff
@@ -10,27 +10,27 @@
 #include "protocol.h"
 
 // Keep track of the active Masternode
-CActiveMasternode activeMasternode;
+CActiveMasternodeInfo activeMasternode;
```

Comment:
Should probably change the name to `activeMasternodeInfo` for it to be a bit more descriptive.

Reply:
Hmm agree. I tried to reduce the number of changed lines so I left this out :see_no_evil: 
Adding new commit on top for this.